vendor/vendor.json

```diff
@@ -1401,5 +1419,5 @@
 			"revision": ""
 		}
 	],
-	"rootPath": "github.com/generationtux/brizo"
+	"rootPath": "brizo"
```

Comment:
Didn't realize it got changed. I will update. Could this have been a govendor issue?